### DISCUSSION TITLE
https://angular.love/angular-facade-pattern?utm_campaign=angular-facade-pattern&utm_medium=newsfeed&utm_source=linkedin&utm_term=eng

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#344] [General]
# https://angular.love/angular-facade-pattern?utm_campaign=angular-facade-pattern&utm_medium=newsfeed&utm_source=linkedin&utm_term=eng

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/angular-facade-pattern?utm_campaign=angular-facade-pattern&utm_medium=newsfeed&utm_source=linkedin&utm_term=eng

**Comment by SuetamId** [comment]
That's really useful content, I've never hearded about it before, thanks for writting! ❤️
